components/Layout.tsx

```diff
@@ -1,50 +1,83 @@
+// react imports
```

Comment:
Just a note that we should set some kind of coding standard for these comments to keep things consistent across files